src/RemoteInboxNotifications/RemoteInboxNotificationsEngine.php

```diff
@@ -66,8 +58,10 @@ public static function on_admin_init() {
 			return;
 		}
 
-		add_action( 'activated_plugin', array( __CLASS__, 'run' ) );
-		add_action( 'deactivated_plugin', array( __CLASS__, 'run_on_deactivated_plugin' ), 10, 1 );
+		if ( 'yes' !== get_option( 'woocommerce_show_marketplace_suggestions' ) ) {
```

Comment:
I think we are okay to remove these two action hooks, it will definitely result in some notes appearing a day later. cc @pmcpinto